docs/installing-on-gke.md

```diff
@@ -0,0 +1,321 @@
+
+In this tutorial you will learn how to deploy and configure OSDFIR Infrastructure
+on Google Kubernetes Engine (GKE). You will then learn how to configure dfTimewolf
+to process a Google Cloud disk using Turbinia and then import any created timelines
+into Timesketch.
+
+
+GRR is not currently supported in this Helm chart deployment. We are working to
+add GRR support in a future release. In the meantime, you can find a dedicated
+guide for deploying GRR on GKE [here](https://github.com/google/osdfir-infrastructure/tree/main/cloud).
+
+## Step 1: Set up Environment Variables
+
+Before creating the Kubernetes (K8s) cluster, define the following environment
+variables in your terminal. Replace the placeholders with your actual values:
+
+```bash
+export PROJECT_ID="your-gcp-project"  # Your Google Cloud project ID
+export PROJECT_NUMBER="your-gcp-number"  # Your Google Cloud project number
+export REGION="us-central1" # The region of your cluster 
+export ZONE="us-central1-f"  # The zone where you want to create the cluster
+export CLUSTER="osdfir-cluster"  # The name you choose for your K8s cluster
+export NAMESPACE="default"  # Your K8s namespace (can be left as 'default')
+export KSA_NAME="turbinia"  # Your Turbinia K8s service account (defaults to 'turbinia' if not set)
+```
+
+> *Note*: You can find the GCP project number by running `gcloud projects describe $PROJECT_ID`
+
+## Step 2: Create a Kubernetes Cluster
+
+Now, create the Kubernetes cluster with the specified configurations:
+
+```bash
+gcloud container clusters create $CLUSTER \
+    --num-nodes=1 \
+    --machine-type "e2-standard-4" \
+    --zone $ZONE \
+    --workload-pool=$PROJECT_ID.svc.id.goog \
+    --addons GcpFilestoreCsiDriver
+```
+
+> *Note*: It will take 4-5 minutes to create the cluster.
+
+This command creates a single-node cluster with the necessary resources and addons
+for running OSDFIR Infrastructure.
+
+Important Considerations:
+
+* GKE Autopilot is not currently supported because Turbinia workers require
+elevated privileges for disk processing.
+* You need a machine type of at least `e2-standard-4` (or equivalent with at
+least 4 CPUs) when deploying to GKE.
+* For clusters with more than one node, you'll need to set up a shared filesystem
+like GCP Filestore. In Kubernetes, this translates to using a Persistent Volume
+Claim (PVC) with `ReadWriteMany` access.
+
+### Configure kubectl to Access the Cluster
+
+Once the cluster has been created, set up the Google Kubernetes Engine auth
+plugin for kubectl:
+
+```bash
+gcloud components install gke-gcloud-auth-plugin
+export USE_GKE_GCLOUD_AUTH_PLUGIN=True
+gcloud container clusters get-credentials $CLUSTER --zone $ZONE
+```
+
+Now check that you can connect to the cluster:
+
+```bash
+kubectl get nodes -o wide
+```
+
+## Step 3: Create the Turbinia GCP Service Account
+
+To process virtual machine disks in Google Cloud Platform (GCP) with Turbinia,
```

Comment:
You need to create the "turbinia" service account first as it does not exist!

Reply:
See my other note about forgetting to update the SA to remove the annotation we set which is not needed anymore but was causing issues with looking for the Turbinia GCP service account